release_management/create_release.py

```diff
@@ -425,6 +429,14 @@ def _get_parser():
         help="stop before pushing any changes to GitHub repo",
     )
 
+    parser.add_argument(
+        "--git-repo",
+        action="store_true",
```

Comment:
Remove the action. It will use the default action, which is "store".

Reply:
Resolved.